.github/workflows/main.yml

```diff
@@ -17,6 +17,24 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
+      - name: Style Check
```

Comment:
Can you put info on how to run this manually in the contributing guidelines as well?